scripts/src/machine.lua

```diff
@@ -1019,6 +1019,19 @@ if (MACHINES["CS8221"]~=null) then
 	}
 end
 
+---------------------------------------------------
+--
+--@src/devices/machine/cs8221.h,MACHINES["CS8900A"] = true
```

Comment:
The header path/name for triggering this with a `SOURCES=` build needs to be correct.